src/models/Stall.ts

```diff
@@ -1,16 +1,25 @@
-import {Model, DataTypes} from 'sequelize';
+import {Model, DataTypes, HasMany, BelongsTo, BelongsToGetAssociationMixin} from 'sequelize';
 
 import sequelize from '../db';
 import Product from './Product';
+import HawkerCentre from './HawkerCentre';
 
 class Stall extends Model {
   public id!: number;
   public name!: string;
-  public description!: string;
-  public contactNo!: string;
-  public unitNo!: string;
-  public address?: string;
+  public description!: string | null;
+  public contactNo!: string | null;
   public hawkerCentreId!: number;
+
+  public readonly createdAt!: Date;
+  public readonly updatedAt!: Date;
+
+  public static HawkerCentre: BelongsTo<Stall, HawkerCentre>;
+  public getHawkerCentre!: BelongsToGetAssociationMixin<HawkerCentre>;
+
+  public static Product: HasMany<Stall, Product>;
```

Comment:
The attribute types here are relations in sequlieze?

Reply:
Yeah this will be initialised to `Stall.Product = Stall.hasMany(Product ...)`. Used for testing. Let me update the code.